icons/stackoverflow/stackoverflow-original.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128"><path d="M.07 1H1v1H.07m0 0"/><path style="fill-rule:evenodd;fill:#bbb;fill-opacity:1" d="M98.125 82.582h11.36V128H7.245V82.582h11.36v34.063h79.519Zm0 0"/><path style="fill-rule:nonzero;fill:#f58025;fill-opacity:1" d="m30.984 79.207 55.793 11.719 2.344-11.149-55.789-11.726ZM38.367 52.5l51.684 24.063 4.816-10.333L43.18 42.168Zm14.301-25.344L96.484 63.63l7.297-8.762-43.816-36.469ZM80.953.2 71.801 7l34.027 45.73 9.149-6.8ZM29.965 105.29h56.8V93.934h-56.8Zm0 0"/></svg>
```

Comment:
Can you remove the `width` and `height` attributes for all icons? Iirc it will allow end-users to more easily change the size of the icon